types/artifacts.d.ts

```diff
@@ -179,6 +181,21 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      export interface IFrameElement {
+        /** The `id` attribute of the IFrame. */
+        id: string,
+        /** The `src` attribute of the IFrame. */
+        src: string,
+        /** The IFrame's ClientRect. @see https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect */
+        clientRect: ClientRect,
+        /** The visibility of the IFrame. */
```

Comment:
can you update this to be more specific (e.g. mention the size requirement)

Reply:
Following Paul's pixelArea idea, so no longer applicable.